File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;

        my_string = my_string.replaceAll("[a-z,A-Z]", "");
        String[] arr = new String[my_string.length()];

        for(int i = 0; i <= my_string.length()-1; i++){
            arr[i] = String.valueOf(my_string.charAt(i)) ;
            answer += Integer.parseInt(arr[i]);
        }
        return answer;
    }
}

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.js
function solution(my_string) {
    let answer = 0;
    
    // 문자열에서 알파벳 문자를 제거합니다.
    my_string = my_string.split('').filter(char => !/[a-zA-Z]/.test(char)).join('');
    
    // 남은 문자를 숫자로 변환하여 합산합니다.
    for (let char of my_string) {
        answer += parseInt(char, 10);
    }
    
    return answer;
}
